### PR TITLE
Add proposed review process

### PR DESCRIPTION
**Pull request by virginiascarlett**
A proposed review process for articles that will be posted to the DSG website. The sketch can be redone; I didn't want to fuss over it at this early stage.

Thanks in advance for your feedback!

### CODE REVIEW

File: REVIEW_PROCESS.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+The questions below do not need to be answered line-by-line. Rather, they are meant to convey a general sense of the purpose of that section. It is up to the reviewers to decide how thoroughly they wish to address each section.
+
+## 1. The Big Picture
+
+Is this article appropriate for the Janelia Data Standards project? Is the general premise of the article sound? Is the proposed data standard generalizable beyond that author’s particular use case?
+
+## 2. Technical Choices
+
+Is the proposed standard elegant, straightforward, and focused? Does the author adequately explain the rationale behind the standard? Is the author making any flawed implicit assumptions, either about the problem or about their audience? Does the author provide an implementation and/or example data?
```
Comment (minnerbe) [inline REVIEW_PROCESS.md:31]: It seems to me that all the questions listed here are necessary conditions for positively evaluation of an article, but most of them are up to interpretation of the reviewer. The only question that is pretty unambiguous is the one about implementation and/or example data. In this sense, this question is qualitatively different from all others asked here.
Do we explicitly want to require an implementation and/or example data? I'm not saying we should or shouldn't. I just want to point out that it's not completely clear to me if it's required in the current version.

File: REVIEW_PROCESS.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Introduction
+
+Articles posted to the Janelia Data Standards website should be vetted, authoritative, well-considered, and of high stylistic and technical quality. For this reason, each new article will undergo a review, using GitHub’s PR and review tooling. This process is intended to help the reviewer refine their ideas.
+
+# Review process
+
+Each new article must be approved by at least two reviewers. At least one of those reviewers must be a **full reviewer**. The other reviewer may be either a **full** or **partial** reviewer. A **full reviewer** must complete all *three* sections of the template below. A **partial reviewer** must complete *at least one* section of the template below.
+
+Reviewers are selected by the author. The author may select more than two reviewers, if desired, but only the approval of two reviewers is needed. The review *may* consist of in-line comments, but it *must* include the template.
```
Comment (minnerbe) [inline REVIEW_PROCESS.md:9]: Who decides which reviewers are full or partial? Is it the decision of the author or is it up to the reviewers to coordinate this themselves? (I would suggest that the default should be the latter, while the former could be a means to get at least one full review in the unlikely case that none of the reviewers volunteer to do it.)

File: REVIEW_PROCESS.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+![A sketch of the review process.](review_process_image.png)
+
+---
+
+# Modifying existing articles
+
+Ideas and standards evolve over time, so some articles will need to be modified after publication. The modification history will be noted clearly on the post itself, so that no modification will be silent.
+
+If the author is making a *minor* modification to their own article, they may skip the review process. A minor modification does not change the audience or the use case for the standard, and poses no danger of rendering existing implementations of the standard obsolete.
```
Comment (minnerbe) [inline REVIEW_PROCESS.md:49]: Is an article a standard? The two terms are used interchangeably in this section.

**Review by minnerbe — COMMENTED** [review]
I like the general style of the proposal: it does not over-specify the review process (which fits the overall endeavor) but makes all expectations explicit. I added a few minor comments in places where the expectations are a bit ambiguous in my view, but the big picture is very good. :+1:

File: REVIEW_PROCESS.md (hunk 1 above)
Comment (virginiascarlett) [reply REVIEW_PROCESS.md]: Changed this:

> [the questions] are meant to convey a general sense of the purpose of that section.

To this:

> [the questions] are meant to guide the reviewer in their subjective evaluation of the article's quality.

And added that example data/implementation are "preferred but not required".

File: REVIEW_PROCESS.md (hunk 2 above)
Comment (virginiascarlett) [reply REVIEW_PROCESS.md]: Right before Wednesday's meeting, I changed the proposal to omit the partial reviewer/full reviewer distinction. Now they are both full reviewers, but the proposal emphasizes that the review can be very short.

File: REVIEW_PROCESS.md (hunk 3 above)
Comment (virginiascarlett) [reply REVIEW_PROCESS.md:47]: Upon reflection, I think they are not interchangeable, and they are not used interchangeably, but the distinction is subtle. Article means article, but standard means "the conventions and recommendations at the heart of the article". So the article can be modified for typos, but that doesn't change the standard. And for example, the standard can be made more permissive, but it's weird to talk about an article becoming more permissive. Perhaps "convention" would be better than "standard"? I am open to suggestions.

Comment (minnerbe) [reply REVIEW_PROCESS.md:39]: So, in your view, a "standard" is a concept and an "article" is *the* written representation of this concept, is this correct?
I understood standard as (an idea in the form of) a document that is separate from articles posted here. In that sense, a standard *is* an article for me and I misunderstood this section. But I agree, the semantics are pretty subtle here.

File: REVIEW_PROCESS.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+# Review template
+
+The questions below do not need to be answered line-by-line. They are not a checklist; rather, they are meant to guide the reviewer in their subjective evaluation of the article's quality. It is up to the reviewer to decide how thoroughly they wish to address each section.
+
+![Image of template.](template_image.png)
+
+## 1. The Big Picture
+
+Is this article appropriate for the Janelia Data Standards project? Is the general premise of the article sound? Is the proposed data standard generalizable beyond that author’s particular use case?
```
Comment (avweigel) [inline REVIEW_PROCESS.md:32]:
```suggestion
Is this article appropriate for the Janelia Data Standards project? 
1. Yes
2. No
3. Comments: ...
Is the general premise of the article sound?
1. Yes
2. No
3. Comments: ...
Is the proposed data standard generalizable beyond that author’s particular use case?
1. Yes
2. No
3. Comments: ...
```

File: REVIEW_PROCESS.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+## 2. Technical Choices
+
+Is the proposed standard elegant, straightforward, and focused? Does the author adequately explain the rationale behind the standard? Is the author making any flawed implicit assumptions, either about the problem or about their audience? Does the author provide an implementation and/or example data (preferable but not required)?
```
Comment (avweigel) [inline REVIEW_PROCESS.md:36]:
```suggestion
Is the proposed standard elegant, straightforward, and focused?
1. Yes
2. No
3. Comments: ...
Does the author adequately explain the rationale behind the standard?
1. Yes
2. No
3. Comments: ...
Is the author making any flawed implicit assumptions, either about the problem or about their audience?
1. Yes
2. No
3. Comments: ...
Does the author provide an implementation and/or example data (preferable but not required)?
1. Yes
2. No
3. N/A
4. Comments: ...
```

File: REVIEW_PROCESS.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+## 3. Writing Style
+
+Is the post readable to bio-imaging developers who may come from a different sub-field, or code in a different language? Does it contain any typos or awkward sentences? Are the ideas organized in a logical flow?
```
Comment (avweigel) [inline REVIEW_PROCESS.md:40]:
```suggestion
Is the post readable to bio-imaging developers who may come from a different sub-field, or code in a different language?
1. Yes
2. No
3. Comments: ...
Does it contain any typos or awkward sentences?
1. Yes
2. No
3. Comments: ...
Are the ideas organized in a logical flow?
1. Yes
2. No
3. Comments: ...
```

**Review by minnerbe — APPROVED** [review]
I think the proposal is great as it is!

**Pull request merged**